administration/src/errors/GraphQlErrorMap.tsx

```diff
@@ -161,9 +161,20 @@ const graphQlErrorMap = (extensions?: ErrorExtensions): GraphQLErrorMessage => {
         title: i18next.t('errors:freinetPersonDataInvalid'),
       }
     }
-    default:
+    case GraphQlExceptionCode.NotImplemented: {
       return {
-        title: defaultError.title,
+        title: i18next.t('errors:functionNotAvailable'),
+      }
+    }
+    case GraphQlExceptionCode.ProjectNotFound: {
+      return {
+        title: i18next.t('errors:projectNotFound'),
```

Comment:
we didn't have an error message for "project not found" before and displayed "seite nicht gefunden"